tests/constants.js

```diff
@@ -36,49 +37,112 @@ export const expectedLayerTreesSelectors = [...baseMetadataSelectors];
 export const expectedStylesSelectors = [...baseMetadataSelectors];
 export const expectedViewsSelectors = [...baseMetadataSelectors];
 
+export function getDeepValues(obj) {
```

Comment:
either don't export it or move it from constants

Reply:
export removed